poi.config.js

```diff
@@ -0,0 +1,32 @@
+const rules = [{
+  test: /bootstrap-sass[/\\]assets[/\\]javascripts[/\\]/,
+  loader: 'imports-loader?jQuery=jquery'
+}];
+
+const port = process.env.PORT || 3000;
+const devServer = {
+  proxy: { '/': `http://localhost:${port}` }
+};
+
+module.exports = options => ({
+  entry: [
+    require.resolve('bootstrap-loader'),
+    'client/main.js'
+  ],
+  preset: [
+    require('poi-preset-buble')(),
+    require('poi-preset-eslint')({ mode: '*' })
```

Comment:
Buble preset has turned `dangerousForOf` transform on by default.
Using wildcard here will include `eslint-loader` in both development and production mode.

Reply:
Great, appreciate the clarification :wink: